README.md

```diff
@@ -74,39 +44,176 @@ This program was created with the following tech stack and packages:
+
+### API
+
+When the API receives a request, the uploaded files are stored in the `/tmp` directory. The API code then reads these inputs and converts them to the appropriate format for Rhubarb and ffmpeg. Upon conversion, the animation process is started by sending all user inputs to the `main.ts` file. 
+
+Here's a typical example of the JSON sent in a POST request.
+
+```markdown
+{
+  fieldname: 'audio',
+  originalname: 'en-Amber.wav',
+  encoding: '7bit',
+  mimetype: 'audio/wave',
+  destination: './tmp',
+  filename: 'f8fc3bc5c3f0fbc70ab0899d0dc0c860',
+  path: 'tmp\\f8fc3bc5c3f0fbc70ab0899d0dc0c860',
+  size: 9049278
+} {
+  fieldname: 'text',
+  originalname: 'en-text.txt',
+  encoding: '7bit',
+  mimetype: 'text/plain',
+  destination: './tmp',
+  filename: 'e05b5794e54ca3810b071b87bc1449e1',
+  path: 'tmp\\e05b5794e54ca3810b071b87bc1449e1',
+  size: 1363
+} phonetic
+```
+
+#### POST Request Properties
+
+The SPP API is the first step in the animation process. Kukurella requests are sent to the SPP API as a POST with the
+following required properties:
+
+| Field      | Type | Description     |
+| :---:       |    :----:  |        :---   |
+| `audio`      | Wav file       | A file containing audio of the voice the user wishes to use for the generated mp4 output |
+| `text`   | Txt file       | A file containing a textual script of what is being communicated in the Wav file      |
+| `recognizer`   | String        | The recognizer for the language processor. Options include default as `english` and non-default `phonetic`. The English-only recognizer is slower but more accurate. The phonetic recognizer is faster but not as accurate, and it supports over 200 languages.      |
+| `avatar` |  String  | Specifies  avatar animation choice for video output. Options include default `barb` or non-default `boy` |
+
+#### API Constraints
+
+- This API only responds to POST calls
+- This process is computationally intensive, which may cause delay between requests. More user requests will result
+in slower processing times
+- The script text file should match the given audio file as closely as possible in terms of content. Failure to do so
+could lead to errors.
+- Port 3000 must be referenced in addition to the IP address.
+- File creation permissions must be allowed, or new files will not be created in `/tmp` directory.
+
+#### Possible API Errors
+
+| Error Code | Description |
+| :---      | :-----     |
+| 400 Bad Request | Required fields are specified |
+| 403 Forbidden | Request is recognized by the server but refused. Likely due to connecting to the wrong port |
+| 500 Internal Server Error | The API has not been configured properly on the host server machine |
+
+### The Phoneme Factory
+
+The Phoneme Factory is the first step in the SPP animation process. The factory is in
+charge of mapping spoken language from the `audio` and `text` inputs into a series of phonemes, which are units of sound
+that distinguish one word from another.
+
```

Comment:
We should update to align with the description from the overview...

Reply:
Resolving, as it was addressed in another commit